recipes/sphinx-jupyterbook-latex/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "sphinx-jupyterbook-latex" %}
+{% set pip_name = name.replace("-", "_") %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ pip_name }}/{{ pip_name }}-{{ version }}.tar.gz
+  sha256: 06d6c15f5ee572d4852421fb9e94ed1dcfd0ff68e03eb0ba61c2829b8114ac8e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - sphinx >=3,<4
+    - myst-nb >=0.11,<0.13
+    - sqlalchemy ~=1.3.12
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+  imports:
+    - sphinx_jupyterbook_latex
+    - sphinx_jupyterbook_latex.theme
+
+
+about:
+  home: "https://github.com/executablebooks/sphinx-jupyterbook-latex"
```

Comment:
none of these quotes are necessary

Reply:
Done in https://github.com/conda-forge/staged-recipes/pull/15234/commits/7b9c4049f152539d813b8c4bf547b77459ba6f8b